app/views/users/show.html.haml

```diff
@@ -0,0 +1,29 @@
+.col-sm-3.user-profile
+  .verba-brand-side
+    = image_tag("https://d1015h9unskp4y.cloudfront.net/attachments/bae8bafd-4081-4b2b-b6f4-c9d4c367ded8/verba.png", class: "verba-logo-small")
+
+  .username
+    = @user.username
+
+  .stats-container
+    .stat
+      .number
+        = @total_posts.count
```

Comment:
Let's just change this to the word count for today's post. The number of posts will be similar to the points, if not the same, so it may be a redundant stat.